platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -31,11 +31,23 @@ MEMORY
 #if defined(S_CODE_SRAM_ALIAS_BASE)
   CODE_RAM (rwx) : ORIGIN = S_CODE_SRAM_ALIAS_BASE, LENGTH = TOTAL_CODE_SRAM_SIZE
 #endif
-#ifndef TFM_MULTI_CORE_TOPOLOGY
+#if !defined(TFM_MULTI_CORE_TOPOLOGY) && !defined(CMSE_VENEER_REGION_IN_CODE)
```

Comment:
I would leave the first #ifdef as is, and then add a nested #ifdef with an explicit comment that elaborates why we do not need a liner area for VENEERS

Reply:
Done